errors/errors.go

```diff
@@ -19,7 +19,7 @@ const (
 	CodeTaskfileCacheNotFound
 	CodeTaskfileVersionCheckError
 	CodeTaskfileNetworkTimeout
-	CodeTaskfileDuplicateInclude
+	_ // CodeTaskfileDuplicateInclude
```

Comment:
We could remove this unused line.

Reply:
Because of how `iota` works, removing this line would change the number assigned to `CodeTaskfileCycle`. It's unlikely that anyone is checking against this specific code in scripts, but if they are then removing this line would break it. I feel like its safer to just deprecate this error. We can maybe assign a different error to it in the future.

Reply:
Got it. I believe we can just keep it for now 👍